Style.md

```diff
@@ -49,25 +72,26 @@
 #### For loops
 
   - Iterator variable should be declared inside the `for` parentheses, unless already defined
+  - Iterator variables should be named `i` if possible. Nested `for` loops use `j`, `k`, etc.
   - Use `for` with arrays, `for...in` for objects (and always check `hasOwnProperty()`)
-  - When iterating over a fixed or calculated size, assign size to a variable inside the `for` declaration and use iterator variable name with `l` suffix
   - Always `++i`, never `i++`
 
   ```javascript
   // Right
 
-  var name = 'john';
+  const name = 'john';
 
-  for (var i = 0, il = name.length; i < il; ++i) {
+  for (let i = 0; i < name.length; ++i) {
       console.log(name[i]);
   }
 
   // Wrong
 
-  var position;
-  var name = 'john' ;
+  let position;
+  const name = 'john' ;
+  const len = name.length;
```

Comment:
Maybe since we're ditching caching of lengths in for loops, show this a different way? It's a little confusing to say "don't cache lengths" then show an example of caching lengths.